app/presenters/two_factor_auth_code/totpable.rb

```diff
@@ -0,0 +1,18 @@
+module TwoFactorAuthCode
+  module Totpable
+    def auth_app_fallback_link
+      t('links.phone_confirmation.auth_app_fallback_html', link: auth_app_fallback_tag)
+    end
```

Comment:
The Code Climate browser extension is showing that this is not covered by tests.